modules/gdscript/tests/scripts/parser/features/export_enum.gd

```diff
@@ -1,15 +1,13 @@
-@export_enum("Red", "Green", "Blue") var untyped
```

Comment:
Might be good to add an `errors` test too with the expected failures for a few examples.